mobile/lib/features/trade/trade_bottom_sheet.dart

```diff
@@ -30,11 +30,13 @@ tradeBottomSheet({required BuildContext context, required Direction direction})
                 currentFocus.unfocus();
               }
             },
-            child: SizedBox(
-                // TODO: Find a way to make height dynamic depending on the children size
-                // This is needed because otherwise the keyboard does not push the sheet up correctly
-                height: 450,
-                child: TradeBottomSheet(direction: direction)),
+            child: SingleChildScrollView(
+              child: SizedBox(
+                  // TODO: Find a way to make height dynamic depending on the children size
+                  // This is needed because otherwise the keyboard does not push the sheet up correctly
+                  height: 450,
+                  child: TradeBottomSheet(direction: direction)),
+            ),
```

Comment:
> ❓Why do we not use a ScrollableSafeArea here?

I did try to use it here but it causes the TradeBottomSheet to expand to fill the entire screen (not what we want), and we don't need it here anyway. First of all, this wasn't using a safe area anyway. Second of all, ScrollableSafeArea is mostly SafeArea + Scroll + some hacks to make things work inside scroll. Since this is already sized, we don't need those hacks, so we just need scroll. That's why SingleChildScrollView fits perfectly